FEATURES.md

```diff
@@ -266,28 +267,35 @@ Floating point numbers must have the decimal point `.` and can optionally take a
 The three number types are fundamentally different.
 If you mix two numbers of different types HVM will interpret the binary representation of one of them incorrectly, leading to incorrect results. Which number is interpreted incorrectly depends on the situation and shouldn't be relied on for now.
 
-At the moment Bend doesn't have a way to convert between the different number types, but it will be added in the future.
-
-You can use `switch` to pattern match on unsigned native numbers:
+Bend now has a way to convert between the different number types!
+Here's some of the builtin functions you can use to cast any native number into the corresponding type:
 
 ```py
-switch x = 4:
-  # From '0' to n, ending with the default case '_'.
-  case 0:  "zero"
-  case 1:  "one"
-  case 2:  "two"
-  # The default case binds the name <arg>-<n>
-  # where 'arg' is the name of the argument and 'n' is the next number.
-  # In this case, it's 'x-3', which will have value (4 - 3) = 1
-  case _:  String.concat("other: ", (String.from_num x-3))
-```
+def main() -> _:
+  x = f24/to_i24(1.0)
+  y = u24/to_f24(2)
+  z = i24/to_u24(-3)
 
+  return (x, y, z)
+```
+You can find the other casting functions and their declarations at [builtins.md](docs/builtins.md).
 ### Other builtin types
 
 Bend has Lists and Strings, which support Unicode characters.
 
+# These are the definitions of the builtin types.
```

Comment:
`#` means a BIG header, better to write something like "Bend has Lists and Strings, ... This is how they're defined."